src/plugins/console/server/lib/spec_definitions/json/overrides/esql.query.json

```diff
@@ -0,0 +1,26 @@
+{
+  "esql.query": {
+    "data_autocomplete_rules": {
+      "columnar": false,
+      "locale": "",
+      "params": [],
+      "query": ""
+    },
+    "url_params": {
+      "format": [
+        "cbor",
+        "csv",
+        "json",
+        "smile",
+        "text",
+        "tsv",
+        "yaml"
+      ],
+      "delimiter": "",
```

Comment:
I think we don't need to add the `delimiter` parameter here as it is already defined in the [generated spec definitions ](https://github.com/elastic/kibana/blob/9faf2f6ebf06941445a05e2a9b63ea374cbcb7f8/src/plugins/console/server/lib/spec_definitions/json/generated/esql.query.json#L9).

Reply:
Tested and indeed we can remove it - done!